src/build.ts

```diff
@@ -3,24 +3,30 @@
 import { IOType } from 'child_process';
 import { join } from 'path';
 
+import { runCommand as runStableBuildCommand } from '#commands/build/index';
```

Comment:
All experimental commands should have a name with Experimental in them, and all stable commands should not have the word Stable or Experimental in them, double-check

Reply:
done